src/pages/faq.tsx

```diff
@@ -0,0 +1,64 @@
+import { Collapse, Text } from '@geist-ui/core';
+import { useRouter } from 'next/router';
+import { ReactMarkdown } from 'react-markdown/lib/react-markdown';
+
+const FAQ = () => {
+  const FAQ = [
```

Comment:
I know it's in different scope, but please consider changing this as it is quite confusing. Your old code is fine, name it FAQs or FAQList.

Reply:
Okay